ddpui/utils/webhook_helpers.py

```diff
@@ -105,9 +114,49 @@ def email_orgusers_ses_whitelisted(org: Org, email_body: str):
         )
 
 
-def email_flowrun_logs_to_orgusers(org: Org, flow_run_id: str):
+def email_flowrun_logs_to_superadmins(org: Org, flow_run_id: str):
     """retrieves logs for a flow-run and emails them to all users for the org"""
     logs_arr = prefect_service.recurse_flow_run_logs(flow_run_id)
     logmessages = [x["message"] for x in logs_arr]
     email_body = generate_notification_email(org.name, flow_run_id, logmessages)
-    email_orgusers(org, email_body)
+    email_superadmins(org, email_body)
+
+
+def notify_org_managers(org: Org, message: str):
+    """send a notification to all users in the org"""
+    error, recipients = get_recipients(
+        SentToEnum.ALL_ORG_USERS, org.slug, None, manager_or_above=True
+    )
+    if error:
+        logger.error(f"Error getting recipients: {error}")
+        return
+    error, response = create_notification(
+        NotificationDataSchema(author="Dalgo", message=message, recipients=recipients)
+    )
+    if error:
+        logger.error(f"Error creating notification: {error}")
+        return
+    logger.info(f"Notification created: {response}")
+
+    if hasattr(org, "preferences"):
+        orgpreferences: OrgPreferences = org.preferences
```

Comment:
Can we remove this ? ITs already being done in `create_notification()`

Reply:
done